rero_ils/modules/loans/api.py

```diff
@@ -706,70 +706,129 @@ def is_notified(self, notification_type=None, counter=0):
         return number_of_reminders_sent(
             self, notification_type=notification_type) > counter
 
-    def create_notification(self, notification_type=None, counter=0):
+    def get_notification_candidates(self, trigger):
+        """Get notifications candidate to create for the notification.
+
+        This function will check the loan and return all possible notifications
+        to create related to it. In some case, a notification must be generate
+        on an other loan than itself (in case of request).
```

Comment:
```suggestion
        This function will check the loans and return all possible notifications
        to be created related to it. In some case, a notification must be generated
        on the basis of another loan than itself (in case of request).
```
Here, "itself" refers to the notification. Maybe "another loan than the first one", or something else?

Reply:
I kept `loan` because we only check 1 loan (itself).